### PR TITLE
Add option to create s3 bucket, as autocreation is depreciated

### PR DESCRIPTION
**Pull request by mrow4a**
@DeepDiver1975 @tboerger @patrickjahns

### CODE REVIEW

**Comment by patrickjahns** [comment]
@mrow4a 
This should not be part of the docker image - creating buckets is a infrastructure task and should be done before starting the image. 

Think of it as the same as with databases - the database on the dbms needs to also exist before ownCloud can be installed. The command in the s3 workspace is for developers as helper, so please do not use it in a production image

**Comment by mrow4a** [comment]
Makes sense @patrickjahns - this command is indeed testing, just checked

However, when I tried to add to `drone.yaml`  command `occ s3:create-bucket owncloud --accept-warning` it failed with permission
```
[create-bucket:L1:0s] + ./occ s3:create-bucket owncloud --accept-warning
[create-bucket:L2:0s] Cannot write into "config" directory!
[create-bucket:L3:0s] This can usually be fixed by giving the webserver write access to the config directory
[create-bucket:L4:0s] 
[create-bucket:L5:0s] See https://doc.owncloud.org/server/10.1/go.php?to=admin-dir_permissions
```

**Comment by tboerger** [comment]
Not entirely sure about this option... At least it can be useful for our own tests like smashbox.

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ OWNCLOUD_MEMCACHED_ENABLED false
 OWNCLOUD_MEMCACHED_HOST memcached
 OWNCLOUD_MEMCACHED_PORT 11211
 OWNCLOUD_OBJECTSTORE_ENABLED false
+OWNCLOUD_OBJECTSTORE_CREATE_BUCKET false
```
Comment (mrow4a) [inline README.md]: Maybe we can mark it as non-production command in README, and not recommended in production environment
? I bet some community users might still use it :> 

@tboerger @patrickjahns @DeepDiver1975

**Comment by patrickjahns** [comment]
I am against using it that way - it only works if `files_primary_s3` has been installed ( and it is not shipped by default ). Additionally - what happens when the container re-starts -> will the command throw errors and bring down the startup process again

**Comment by mrow4a** [comment]
@patrickjahns it will print message that bucket already exists and return.

**Comment by mrow4a** [comment]
Closed with https://github.com/owncloud/smashbox-testing/pull/2

**Pull request closed without merging**